packages/manager/apps/pci-vouchers/src/pages/AddVoucherPage.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import { render } from '@testing-library/react';
+import { describe, it, vi } from 'vitest';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import {
+  ShellContext,
+  ShellContextType,
+} from '@ovh-ux/manager-react-shell-client';
+import AddVoucherPage from './AddVoucherPage';
+
+vi.mock('react-router-dom', () => ({
+  useParams: () => ({ projectId: '123' }),
+  useNavigate: () => vi.fn(),
+}));
+
+vi.mock('@ovhcloud/manager-components', () => ({
+  useNotifications: () => vi.fn(),
+}));
+
+vi.mock('react-i18next', () => ({
+  useTranslation: () => ({ t: (key) => key }),
+}));
+
```

Comment:
already defined in `packages/manager/apps/pci-vouchers/src/setupTests.ts` so no need to define it one more time

Reply:
done